README.md

```diff
@@ -2,7 +2,16 @@
 
 ## Descripton
 
-[cstate](https://github.com/cstate/cstate) is a powerful tool to allow displaying of system status.  However, being a statically rendered site, it doesn't include any native healthcheck capabilities.  This project provides that capability.  It uses AWS Route53 HealthChecks to automatically create new outages notifications in your cstate github repository, and resolve them when the service is available.
+[cstate](https://github.com/cstate/cstate) is a powerful tool to allow displaying of system status.  However, being a statically rendered site, it doesn't include any native healthcheck capabilities.  This project provides that capability.  It uses AWS Route53 HealthChecks to automatically create new outage notifications in your cstate github repository, and resolve them when the service becomes available.
+
+## Features
+
+* Serverless (AWS Lambda)
+* Configuratble ability to ignore short downtime if desired
```

Comment:
I think I would rewrite this as something like:
`Ability to ignore downtime for a desired duration, e.g. ignore downtime of 5 min or less`